src/Groups/group_characters.jl

```diff
@@ -187,12 +270,12 @@ nothing
 ```
 """
 function character_table(id::String, p::Int = 0)
-    hasproperty(GAP.Globals, :CTblLib) || error("no character table library available")
+    @req hasproperty(GAP.Globals, :CTblLib) "no character table library available"
```

Comment:
`@req` throws an argument error, that doesn't seem quite appropriate here?

Also, aren't we loading ctbllib during startup? (Though, I guess, we are not checking whether that worked... perhaps we should?

Reply:
Yes, we call `GAP.Packages.load` with `"ctbllib"` and several other names of GAP packages, without looking at the results.
(For `"browse"` we even add the optional argument `install=true`, but again without consequences if this fails.)
This means "load this GAP package if it is available, but do not complain if it is not available".
If we are sure that no Oscar users will run into problems because of missing GAP packages then we can check that the `GAP.Packages.load` calls return `true`, and then we can rely on the packages' availability.

Concerning `@req`:
Yes, I had also thought in which cases `ArgumentError` is the right reaction, but came to the conclusion that any error that occurs inside a function can be regarded as the consequence of a problem with the arguments.
(Well, there could also be a bug in the code of the function, but typically such bugs are not detected with error messages.)
In the situation in question, using a string as the first argument of `character_table` is not supported if the character table library is not available.

Reply:
but if the character table library is missing then that isn't an error in the arguments, at least if the exact same input works on a different computer